zerver/migrations/0521_set_userpresence_last_update_id_not_null.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.0.5 on 2024-05-02 21:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0520_user_presence_backfill_last_update_id_to_0"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="userpresence",
+            name="last_update_id",
+            field=models.PositiveBigIntegerField(db_index=True, default=0),
+        ),
```

Comment:
This operation could probably be in migration 0520, since it's non-atomic anyway?